common/djangoapps/django_comment_common/models.py

```diff
@@ -150,8 +151,15 @@ def all_permissions_for_user_in_course(user, course_id):  # pylint: disable=inva
 class ForumsConfig(ConfigurationModel):
     """Config for the connection to the cs_comments_service forums backend."""
 
-    # For now, just tweak the connection timeout settings. We can add more later.
-    connection_timeout = models.FloatField(default=5.0)
+    connection_timeout = models.FloatField(
+        default=5.0,
+        help_text="Seconds to wait when trying to connect to the comment service.",
+    )
+
+    @property
+    def api_key(self):
+        """The API key used to authenticate to the comments service."""
+        return getattr(settings, "COMMENTS_SERVICE_KEY", None)
```

Comment:
Just for proof:

![image showing only the connection timeout field, and a lack of the API key field](https://i.imgur.com/FHs4wBD.png)